src/material/schematics/ng-add/setup-project.ts

```diff
@@ -72,7 +72,7 @@ function addAnimationsModule(options: Schema) {
     try {
       addAnimationsModuleToNonStandaloneApp(host, project, context, options);
     } catch (e) {
-      if (e instanceof SchematicsException && e.message.includes('Bootstrap call not found')) {
+      if (e instanceof Error && e.message.includes('Bootstrap call not found')) {
```

Comment:
I made this change, because I ended up with a version mismatch between some of the Tooling packages by accident which broke the check here. Checking against `Error` instead should be more robust.